### PR TITLE
wrote some queries

### PR DESCRIPTION
**Pull request by Ntare22**
mainly practiced with the SELECT and queried different things from the DB

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,17 +15,12 @@ This repository includes files with plain SQL that can be used to recreate a dat
```
Comment (SajjadAhmad14) [inline README.md:13]: Please add a project title and features description in README as it looks more professional.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,35 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+/* Find all animals whose name ends in "mon". */
+
+SELECT * from animals WHERE name LIKE '%mon%';
```
Comment (SajjadAhmad14) [inline queries.sql]: Please add a query to find all animals that have names ending with "mon" not starting and ending with "mon".

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Jim, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by oloomoses — APPROVED** [review]
Hi @Ntare22, 

Well done on making the required changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**